jax/experimental/jax2tf/jax_export.py

```diff
@@ -83,6 +83,16 @@ def is_custom_call(self) -> Optional[str]:
     m = re.match(r'custom_call:(.+)$', self._impl)
     return m.group(1) if m else None
 
+  @classmethod
```

Comment:
Is the dialect string case sensitive? Should any normalization be done?

Reply:
It is case sensitive, and should match the dialect name. This should be Ok, because if the user gets the name wrong in the `disabled_checks` they will get an error that the dialect is not allowed.